src/app/GitUI/UserControls/PatchGrid.cs

```diff
@@ -137,7 +137,11 @@ private IReadOnlyList<PatchFile> GetInteractiveRebasePatchFiles()
             }
             else
             {
-                ObjectId.TryParse(parts[1], out objectId);
+                if (!ObjectId.TryParse(parts[1], out objectId))
+                {
+                    Trace.Write($"PatchGrid: GetInteractiveRebasePatchFiles: Unable to parse commit hash '{parts[1]}' from '{todoCommits}'. Skipping this entry.");
```

Comment:
Use `nameof`?

Reply:
This is a printout that "should not happen". The printout should be unique enough.
'{todoCommits}' to "{nameof(todoCommits)}: '{todoCommits}'" is not a big improvement (and the line is already too long)